CLAUDE.md

```
@@ -0,0 +1,333 @@
+# app_name/cores/speech_processor.py
+class SpeechProcessorCore:
+    """پردازش صوت (STT/TTS)"""
+    pass
+
+# app_name/cores/orchestrator.py
+class CentralOrchestrator:
+    """هماهنگی بین هسته‌ها"""
+    pass
+```
+
+### 3. مدل‌های یکپارچه
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)